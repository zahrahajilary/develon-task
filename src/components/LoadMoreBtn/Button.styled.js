import styled from "styled-components";

export const ButtonContainer = styled.div`
  font-family: "Ubuntu", sans-serif;
  position: relative;
  font-weight: 400;
  font-size: 1.3rem;
  line-height: 2;
  height: 50px;
  transition: all 200ms linear;
  border-radius: 4px;
  width: 240px;
  letter-spacing: 1px;
  display: flex;
  align-items: center;
  justify-content: center;
  text-align: center;
  align-self: center;
  border: none;
  cursor: pointer;
  box-shadow: 0 12px 35px 0 rgba(16, 39, 112, 0.25);
  outline: 0;
  text-transform: capitalize;
  align-self: center;
  justify-content:center;
  text-align:center;
  border:1px solid white;
  margin : 16px auto
  
  `

;
