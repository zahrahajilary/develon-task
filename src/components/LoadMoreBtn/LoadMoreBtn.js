import React from 'react';
import {ButtonContainer} from "./Button.styled";

const LoadMoreBtn = ({  onClick, children }) => {
    return (
        <ButtonContainer onClick={()=>onClick()}
        >
            {children}
        </ButtonContainer>
    );
};

export default LoadMoreBtn;
