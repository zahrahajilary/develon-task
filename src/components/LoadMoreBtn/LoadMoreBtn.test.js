
import { shallow } from 'enzyme';
import React from 'react';

import LoadMoreBtn from "./LoadMoreBtn";

it('should render LoadMore component', () => {

    expect(shallow(<LoadMoreBtn onClick={()=>console.log(hi)} >btn</LoadMoreBtn>)).toMatchSnapshot();
});
