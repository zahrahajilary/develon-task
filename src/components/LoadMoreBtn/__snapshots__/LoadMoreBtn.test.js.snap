// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadMore component 1`] = `ShallowWrapper {}`;
